### PR TITLE
[WBCAMS-264] stop greying-out expired jobs

### PR DESCRIPTION
**Pull request by jonathan-longe**
Previously, the job board angular application would grey out job listings that were close to the expiry date.  Now, the job board displays the job listing until the expiry date without greying them out.

### CODE REVIEW

**Comment by jonathan-longe** [comment]
@JonTaylor1971 please hold off reviewing this PR.  I just realized I have further changes to make.

**Comment by jonathan-longe** [comment]
Changes complete.  @JonTaylor1971 I'd appreciate any comments you may have on this pull request especially since you've worked on this ticket previously.

File: src/WorkBC.Web/WorkBC.Web.csproj [hunk 1]
```diff
@@ -66,6 +66,9 @@
     <Content Remove="ClientApp\projects\jb-search\src\app\services\**" />
     <Content Remove="ClientApp\src\**" />
     <None Include="$(SpaRoot)**" Exclude="$(SpaRoot)node_modules\**" />
+    <Compile Remove="wwwroot\**" />
+    <Content Remove="wwwroot\**" />
+    <None Remove="wwwroot\**" />
```
Comment (jonathan-longe) [inline src/WorkBC.Web/WorkBC.Web.csproj:71]: @JonTaylor1971 any thoughts on why this file shows as changed?  I'm considering rolling back the changes.

**Review by JonTaylor1971 — COMMENTED** [review]
Looks good based on our discussion.

**Review by JonTaylor1971 — APPROVED** [review]

**Pull request merged**